app/client/packages/design-system/ads-old/src/EditableTextSubComponent/index.tsx

```
@@ -215,17 +218,31 @@ export const EditableTextSubComponent = React.forwardRef(
         }
 
         setValue(finalVal);
         setIsInvalid(error);
         setChangeStarted(true);
       },
-      [inputValidation, onTextChanged],
+      [inputValidation, onTextChanged, valueTransform],
     );
 
+    const onCancel = useCallback(() => {
+      onBlur && onBlur(lastValidValue);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)